src/topolvm/generate_manifests.sh

```diff
@@ -65,25 +70,48 @@ EOF
         "port": "healthz",
         "path": "/healthz"
       }
-    })' -i "${ASSETS_DIR}/03-topolvm.yaml"
+    })' -i "${ASSETS_DIR}/02-topolvm.yaml"
+  # Generate Patch with annotation to dynamically inject the CA bundle
+  cat >"${ASSETS_DIR}/topolvm_mutatingwebhook_patch.yaml" <<'EOF'
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+metadata:
+  name: topolvm-hook
+  annotations:
+    service.beta.openshift.io/inject-cabundle: "true"
+webhooks:
+  - name: pvc-hook.topolvm.io
+EOF
+
+  cat >"${ASSETS_DIR}/topolvm_service_patch.yaml" <<'EOF'
+apiVersion: v1
+kind: Service
+metadata:
+  name: topolvm-controller
+  namespace: topolvm-system
+  annotations:
+    service.beta.openshift.io/serving-cert-secret-name: topolvm-mutatingwebhook
+EOF
 
-  # Generate kustomize
+# Generate kustomize
   cat >"${ASSETS_DIR}/kustomization.yaml" <<'EOF'
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
   - 01-namespace.yaml
-  - 02-cert-manager.yaml
-  - 03-topolvm.yaml
+  - 02-topolvm.yaml
+patches:
+  - path: topolvm_mutatingwebhook_patch.yaml
+  - path: topolvm_service_patch.yaml
 EOF
 }
 
 generate_release_file() {
   local -r rel_arch=$1
   local -r rel_file="${RELEASE_DIR}/release-topolvm-${rel_arch}.json"
-  local -r tmp_file=$(mktemp /tmp/topolvm-release-XXXXXX)
+  local -r tmp_file=$(mktemp /tmp/topolvm-release-XXXXXX.txt)
```

Comment:
We had PR from @agullon removing `.txt` extension. 
Do we need to add it here?

See https://github.com/microshift-io/microshift/pull/144